jobs/newrelic-monitor/spec

```diff
@@ -15,3 +15,6 @@ properties:
     description: "Newrelic License Key"
   newrelic.deployment_tag:
     description: "The deployment tag that would appear in New Relic."
+  newrelic.proxy:
+    description: "http proxy to use to access newrelic saas "
```

Comment:
maybe precise that this will be used for both http and https connections 

description: "http/https proxy to use to access newrelic saas "

and potentially link to newrelic documentation at https://docs.newrelic.com/docs/servers/new-relic-servers-linux/installation-configuration/linux-configuration-new-relic-servers (or duplicate it) to clarify the proxy format

> Certain sites have very restrictive egress firewalls and require you to use a proxy in order to communicate with the outside world. If this applies to your site, you will need to set this value in the form user:password@hostname:port.
> 
> Several portions of this value are optional, depending on your proxy configuration.
> 
>    If your proxy does not require a user name and password, use hostname:port.
>    If your proxy is using the default port for that proxy, the port is optional. For most proxies the default is 1080.
>    The default proxy is http. Other proxies should be specified, For example, if the proxy is socks/socks5h, specify it before the hostname: socks5h://hostname:port